src/App/Documentation/components/Forms/index.js

```diff
@@ -10,7 +10,11 @@ import { Checkbox, FormControlText, Radio, Rangeslider, Togglebox } from "@/Form
 const Overview = () => (
     <>
         <h2 id="overview">Overview</h2>
-        <p>Forms overview...</p>
+        <p>
+            A group of forms must be wrapped contained by a form element. Each input element must be wrapped by its own form-group, but every form-group can contain a number of elements.
```

Comment:
This sentence is a bit tangled up. Perhaps this is an improvement? I'm also not sure what it states is correct.

```suggestion
            A group of form elements must be contained in a form element. Each input element must be wrapped by its own form-group, but every form-group can contain a number of elements.
```

Reply:
Changed all of the text as a big part of it was wrong..